pkg/controller/expand_and_recover.go

```diff
@@ -90,32 +88,31 @@ func (ctrl *resizeController) expandAndRecover(pvc *v1.PersistentVolumeClaim, pv
 		//      safe to do so.
 		//   4. While expansion was still pending on the node, user reduced the pvc size.
 		switch resizeStatus {
-		case v1.PersistentVolumeClaimNodeExpansionInProgress,
-			v1.PersistentVolumeClaimNodeExpansionPending:
+		case v1.PersistentVolumeClaimNodeResizeInProgress,
+			v1.PersistentVolumeClaimNodeResizePending:
 			// we don't need to do any work. We could be here because of a spurious update event.
 			// This is case #1
 			return pvc, pv, nil, resizeNotCalled
-		case v1.PersistentVolumeClaimNodeExpansionFailed:
+		case v1.PersistentVolumeClaimNodeResizeFailed:
 			// This is case#3, we need to reset the pvc status in such a way that kubelet can safely retry volume
 			// expansion.
 			if ctrl.resizer.DriverSupportsControlPlaneExpansion() && allocatedSize != nil {
 				newSize = *allocatedSize
 			} else {
 				newSize = pvcSpecSize
 			}
-		case v1.PersistentVolumeClaimControllerExpansionInProgress,
-			v1.PersistentVolumeClaimControllerExpansionFailed,
-			v1.PersistentVolumeClaimNoExpansionInProgress:
+		case v1.PersistentVolumeClaimControllerResizeInProgress,
+			v1.PersistentVolumeClaimControllerResizeFailed:
 			// This is case#2 or it could also be case#4 when user manually shrunk the PVC
 			// after expanding it.
 			if allocatedSize != nil {
 				newSize = *allocatedSize
 			}
 		default:
-			// It is impossible for ResizeStatus to be nil and allocatedSize to be not nil but somehow
+			// It is impossible for ResizeStatus to be empty and allocatedSize to be not nil but somehow
```

Comment:
Nit: the comment here is different from the other PR https://github.com/kubernetes/kubernetes/pull/116335. Should this be empty or ""?

Reply:
empty and "" are same thing right? the main thing is - corresponding intree code doesn't ever actually gets executed because all volume plugins that support volume expansion are actually now migrated to CSI.